src/openforms/registrations/contrib/email/plugin.py

```diff
@@ -77,51 +73,64 @@ def render_registration_email(submission, extra_context=None):
         }
         base_context = {
             "public_reference": submission.public_registration_reference,
-            "datetime": timezone.localtime(submission.completed_on).strftime(
+            "completed_on": timezone.localtime(submission.completed_on).strftime(
```

Comment:
Love this name change. Does it need a data migration?

Reply:
I don't think so, since up to now the templates were not customisable? :thinking: